lib/command/common/run-with-cover.js

```diff
@@ -140,7 +142,7 @@ function run(args, commandName, enableHooks, callback) {
         excludes.push(path.relative(process.cwd(), path.join(reportingDir, '**', '*')));
         matcherFor({
             root: config.instrumentation.root() || process.cwd(),
-            includes: [ '**/*.js' ],
+            includes: opts.i || [ '**/*.js' ],
```

Comment:
I think you should not use `opts.i` directly here, because then you are not able to configure this option in the `.istanbul.yml` file.